.github/workflows/build-and-push-component.yaml

```diff
@@ -24,34 +20,24 @@ on:
         required: true
         type: string
         description: 'The build timestamp to be used for binaries.'
+      bake_target:
+        required: true
+        type: string
+        description: 'Docker Bake target name.'
+      cache-name-prefix:
+        required: true
+        type: string
+        description: 'A prefix for image names and digests to be cached temporarily.'
+      post_image_tags:
+        required: false
+        type: boolean
+        description: |
+          If set to true the image tags pushed to the repository are posted as comment for the Pull Request.
+          Only works if the event type is `pull_request`.
```

Comment:
Is this really needed for this action? afaik this will never be hit as this is executed manually. only the reusable action requires this

Reply:
CI changes will be part of another PR